apps/mobile/src/components/Feed/Socialize/CommentBox.tsx

```diff
@@ -97,6 +97,7 @@ export function CommentBox({
         <BottomSheetTextInput
           value={value}
           onChangeText={setValue}
+          multiline={true}
```

Comment:
nit: can just be `multiline` (don't need to pass true)

Reply:
yeah totally! 👍